pkg/reconciler/broker/broker.go

```diff
@@ -149,30 +153,16 @@ func (r *Reconciler) reconcileBroker(ctx context.Context, b *brokerv1beta1.Broke
 	b.Status.InitializeConditions()
 	b.Status.ObservedGeneration = b.Generation
 
+	if err := r.ensureBrokerCellExists(ctx, b); err != nil {
+		return fmt.Errorf("brokercell reconcile failed: %v", err)
+	}
```

Comment:
Will the brokercell controller fight with the broker controller? It seems that they will both reconcile data plane deployments?

Reply:
No. The broker controller only creates a `brokercell` resource if needed. The actual reconciliation of data plane is done by the brokercell controller.